From b278dcb03de381c1e08e9d9641394c3dac7116fc Mon Sep 17 00:00:00 2001
From: David Sniatkiwicz <davidsn@marvell.com>
Date: Mon, 28 Aug 2017 17:20:27 +0300
Subject: [PATCH 1222/1345] dts: comphy: a70x0: a80x0: skip PCIe power off for
 A7K-PCAC and A8k-OCP boards

commit  e7d50d4c42a6c3895e277ab27680416160480a36 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Add skip_pcie_power_off flag to pcac and ocp boards.
Since those boards using PCIe as End point,
thus link must be always kept alive.

Change-Id: Ida001b4173e8aa1ab32f4ca9bcffa374d3b00f71
Signed-off-by: David Sniatkiwicz <davidsn@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/43462
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-pcac.dts |    1 +
 arch/arm64/boot/dts/marvell/armada-8040-ocp.dts  |    1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-pcac.dts b/arch/arm64/boot/dts/marvell/armada-7040-pcac.dts
index c01b830..4d6c244 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-pcac.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-pcac.dts
@@ -92,6 +92,7 @@
 	cpn-110-master {
 		config-space {
 			comphy0: comphy {
+				skip_pcie_power_off;
 				status = "okay";
 			};
 			sata: sata@540000 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts b/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts
index e89cd15..08dd997 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-ocp.dts
@@ -94,6 +94,7 @@
 				};
 			};
 			comphy0: comphy {
+				skip_pcie_power_off;
 				status = "okay";
 			};
 			i2c@701000 {
-- 
1.7.9.5

